src/test/kotlin/com/pubnub/api/legacy/endpoints/files/SendFileTest.kt

```diff
@@ -63,20 +51,61 @@ class SendFileTest : TestsWithFiles {
         every { publishFileMessageFactory.create(any(), any(), any()) } returns publishFileMessage
 
         // when
-        val result: PNFileUploadResult? = FileInputStream(file).use { fileInputStream ->
-            sendFile(channel, file.name, fileInputStream).sync()
+        val result: PNFileUploadResult? = inputStream().use { inputStream ->
+            sendFile(channel, fileName(), inputStream).sync()
         }
 
         // then
         Assert.assertEquals(expectedResponse, result)
     }
 
     @Test
-    @Throws(InterruptedException::class, IOException::class)
+    fun spaceIdAndMessageTypeArePassedToPublishMessageFile() {
```

Comment:
Are you sure that test does what it promises? Is this possible to pass spaceId and messageType to publishMessageType?

Reply:
Nope. This shouldn't be there. It's a spoiler from other PR that will come soon ;)

Reply:
ok